scripts/importer_engine_support_check.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Generates configs from the full example, inits all modules, and checks if any resources are not supported by the importer.
+// Meant to be run from the scripts/ folder like so:
+// go run importer_engine_support_check.go
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"sort"
+	"strings"
+
+	"github.com/GoogleCloudPlatform/healthcare-data-protection-suite/internal/tfengine"
+	"github.com/GoogleCloudPlatform/healthcare-data-protection-suite/internal/tfimport"
+)
+
+func main() {
```

Comment:
Add doc for what it checks for?

Reply:
I have it higher up, above the package declaration. Should I put it above `main()` instead?

Reply:
Saw that now. No problem. It's good.